src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -174,6 +174,12 @@ export const buildATIPageTrackPath = ({
       value: isLocServeCookieSet(),
       wrap: true,
     },
+    {
+      key: 'ref',
+      description: 'referrer url',
+      value: getReferrer(platform, origin, previousPath),
```

Comment:
I think it would be useful to decode the referrer before passing it in. See https://github.com/bbc/simorgh/blob/872f7513132fe097c526c5ed961772c73221b1b9/src/app/containers/ChartbeatAnalytics/utils/index.js#L149

Reply:
The latest advice from ATI was to encode this value